rtb-house/DOCUMENTATION.md

```diff
@@ -2,56 +2,106 @@
 ## General Compatibility
 |Feature|  |
 |---|---|
-| Consent |  |
-| Native Ad Support |  |
-| SafeFrame Support |  |
-| PMP Support | |
+| Consent | Yes |
+| Native Ad Support | No |
+| SafeFrame Support | No |
+| PMP Support | No |
  
 ## Browser Compatibility
 | Browser |  |
 |--- |---|
-| Chrome |  |
-| Edge |  |
-| Firefox |  |
-| Internet Explorer 9 |  |
-| Internet Explorer 10 |  |
-| Internet Explorer 11 |  |
-| Safari |  |
-| Mobile Chrome | |
-| Mobile Safari | |
-| UC Browser | |
-| Samsung Internet | |
-| Opera | |
+| Chrome | Yes |
+| Edge | Yes |
+| Firefox | Yes |
+| Internet Explorer 9 | Yes |
+| Internet Explorer 10 | Yes |
+| Internet Explorer 11 | Yes |
+| Safari | Yes |
+| Mobile Chrome | Yes |
+| Mobile Safari | Yes |
+| UC Browser | Yes |
+| Samsung Internet | Yes |
+| Opera | Yes |
  
 ## Adapter Information
 | Info | |
 |---|---|
 | Partner Id | RTBHouseHtb |
-| Ad Server Responds in (Cents, Dollars, etc) | |
-| Bid Type (Gross / Net) | |
-| GAM Key (Open Market) | |
-| GAM Key (Private Market) | |
-| Ad Server URLs | |
-| Slot Mapping Style (Size / Multiple Sizes / Slot) | |
-| Request Architecture (MRA / SRA) | |
+| Ad Server Responds in (Cents, Dollars, etc) | Dollars |
+| Bid Type (Gross / Net) | Net |
+| GAM Key (Open Market) | n/a |
+| GAM Key (Private Market) | n/a |
+| Ad Server URLs | https://prebid-[eu/asia/us].creativecdn.com/bidder/prebid/bids? |
+| Slot Mapping Style (Size / Multiple Sizes / Slot) | Multiple Sizes |
+| Request Architecture (MRA / SRA) | MRA |
```

Comment:
Myślałem o tym jeszcze trochę, zmieniam zdanie. Wolimy powiedzieć SRA.
Będziemy to na pewno kiedyś poprawiać tak, by liczyć takie grube requesty lepiej - im więcej ich będzie tym lepsza motywacja.

Reply:
OK, to przestawiam. W takiej sytuacji przykładowy request i response zmienię, żeby miały po 2 sloty i było widać, że to ta forma.